### PR TITLE
Changes to LineRenderer for Tracers

### PR DESCRIPTION
**Pull request by o0F-0oF**
amongus

### CODE REVIEW

File: KiraiMod/Modules/Tracers.cs [hunk 1]
```diff
@@ -36,12 +36,16 @@ public static void UpdatePlayerTracers()
...
+            lr.material = new Material(Shader.Find("Hidden/Internal-Colored"));
             lr.startWidth = 0.002f;
             lr.endWidth = 0.002f;
             lr.useWorldSpace = false;
             lr.endColor = color;
             lr.startColor = color;
+            lr.sortingOrder = -10;
+            lr.sortingLayerID = 7711475;
+            lr.material.SetInt("_ZWrite", 1.0);
+            lr.material.SetInt("_ZTest", 0.0);
```
Comment (xKiraiChan) [inline KiraiMod/Modules/Tracers.cs]: maybe this should be configurable with a ConfigEntry<bool>

**Review by xKiraiChan — CHANGES_REQUESTED** [review]

File: KiraiMod/Modules/Tracers.cs (hunk 1 above)
Comment (o0F-0oF) [reply KiraiMod/Modules/Tracers.cs]: ok

Comment (o0F-0oF) [reply KiraiMod/Modules/Tracers.cs]: I have completely rewritten that

**Comment by o0F-0oF** [comment]
Rewriting using UnityGL

**Pull request closed without merging**